<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JavaTab</class>
 <widget class="QWidget" name="JavaTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>JVM settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="javaPath_label">
          <property name="text">
           <string>Java Path:</string>
          </property>
          <property name="buddy">
           <cstring>javaPath_le</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="javaPath_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="autoJava_pb">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="javaPath_showPb">
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Arguments Policy</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="officialPolicy_rb">
           <property name="text">
            <string>Offial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="KaminecPolicy_rb">
           <property name="text">
            <string>Kaminec</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="CustomPolicy_rb">
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="customJavaArg_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Custom Java Arguments:</string>
          </property>
          <property name="buddy">
           <cstring>customJavaArg_te</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QTextEdit" name="customJavaArg_te"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Memory Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="minMem_label">
             <property name="text">
              <string>Minimum Memory Allocate:</string>
             </property>
             <property name="buddy">
              <cstring>minMem_sb</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="minMem_sb">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="maxMem_label">
             <property name="text">
              <string>Maximum Memory Allocate:</string>
             </property>
             <property name="buddy">
              <cstring>maxMem_sb</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxMem_sb">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>javaPath_le</tabstop>
  <tabstop>autoJava_pb</tabstop>
  <tabstop>javaPath_showPb</tabstop>
  <tabstop>officialPolicy_rb</tabstop>
  <tabstop>KaminecPolicy_rb</tabstop>
  <tabstop>CustomPolicy_rb</tabstop>
  <tabstop>customJavaArg_te</tabstop>
  <tabstop>minMem_sb</tabstop>
  <tabstop>maxMem_sb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
